docs/fsharp/using-fsharp-on-azure/package-management.md

```diff
@@ -52,16 +52,16 @@ Or, for Mono development:
 If you're using [NuGet](https://www.nuget.org/) as your dependency manager, you can use the `nuget.exe` tool to add Azure dependencies. For example:
```

Comment:
We should be using `dotnet add package`;  there's no need for `nuget.exe`  any longer.  see: [ps://docs.microsoft.com/nuget/consume-packages/install-use-packages-dotnet-cli](ps://docs.microsoft.com/nuget/consume-packages/install-use-packages-dotnet-cli)